wandb/apis/public.py

```diff
@@ -4687,6 +4687,10 @@ def _get_code_artifact(self, artifact_string):
             code_artifact = self._api.artifact(name=artifact_string, type="code")
         if code_artifact is None:
             raise LaunchError("No code artifact found")
+        if code_artifact.state == "DELETED":
```

Comment:
can we use `from wandb.sdk.artifacts.artifact_state import ArtifactState`?
```suggestion
        if code_artifact.state == ArtifactState.DELETED:
```